packages/components/pie-button/src/index.ts

```diff
@@ -1,30 +1,33 @@
 import { LitElement, html, unsafeCSS } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 import { customElement, property } from 'lit/decorators.js';
+
 import styles from './button.scss?inline';
-import { BUTTON_VARIANT, BUTTON_TYPE } from './defs';
-import { validValues } from './decorators';
+import { validPropertyValues } from './decorators';
+import {
+    BUTTON_TYPE,
+    BUTTON_VARIANT,
+    VALID_BUTTON_TYPES,
+    VALID_BUTTON_VARIANTS
+} from './defs';
 
 // Valid values available to consumers
 export { BUTTON_VARIANT, BUTTON_TYPE };
 
-const validButtonVariants = Object.values(BUTTON_VARIANT);
-const validButtonTypes = Object.values(BUTTON_TYPE);
-
 @customElement('pie-button')
 export class PieButton extends LitElement {
     /**
      * The Button type to use
      */
     @property()
-    @validValues(validButtonTypes, BUTTON_TYPE.SUBMIT)
+    @validPropertyValues(VALID_BUTTON_TYPES, BUTTON_TYPE.SUBMIT)
```

Comment:
Is there any other use case for `VALID_BUTTON_TYPES` and `VALID_BUTTON_VARIANTS` that would justify not doing 
`@validPropertyValues(Object.values(BUTTON_TYPE), BUTTON_TYPE.SUBMIT)`?

Reply:
Probably not, I wanted to keep this line as clean as possible but I'll update it